python_modules/dagster/dagster/_config/structured_config.py

```diff
@@ -63,7 +65,12 @@ class Config(MakeConfigCacheable):
     """
 
 
-class PermissiveConfig(Config, extra="allow"):
+class PermissiveConfig(Config):
+    # Pydantic config for this class
+    # Cannot use kwargs for base class as this is not support for pydnatic <1.8
+    class Config:
+        extra = "allow"
```

Comment:
are you confident the subclassing here works the way it did with the kwargs approach

Reply:
Good call. Added unit test.